### PR TITLE
Add compliance section and certification info

### PR DESCRIPTION
**Pull request by yanjost**

### CODE REVIEW

File: src/_posts/compliance/2000-01-01-hds.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
+---
+title: Hébergeur de Données de Santé - Health Data Hosting Provider 
+nav: HDS
+modified_at: 2022-09-12 00:00:00
+tags: compliance hds
+---
+
+*(version française [ci-dessous](./2000-01-01-hds.md#fr-has-scalingo-an-hds-certification))*
```
Comment (btrd) [inline src/_posts/compliance/2000-01-01-hds.md:8]: 404 on the link https://documentation-service-pr1739.osc-fr1.scalingo.io/compliance/hds

File: src/_posts/compliance/2000-01-01-hds.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+3. Provision and maintenance in operational condition of the information system's application hosting platform;
+4. Provision and maintenance in operational condition of the virtual infrastructure of the information system used for processing health data;
+5. Administration and operation of the information system containing the Personal Health Data;
+6. Outsourced backups of health data.
+
+## 🇬🇧 Where can I get the official latest version of your HDS certificate ?
+
+Please always refer the current page to get the latest compliance information
+
+[Download the current HDS 1.1 certificate](https://cdn.scalingo.com/documents/20220912-scalingo-hds-lne-certificate-38436-rev0.pdf)
```
Comment (btrd) [inline src/_posts/compliance/2000-01-01-hds.md:37]: Maybe put an `<hr>` after to divide English part to French part ?

File: src/_posts/compliance/2000-01-01-iso27001.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: "NF EN ISO/CEI 27001:2017"
+nav: ISO 27001
+modified_at: 2022-09-12 00:00:00
+tags: compliance iso-27001
+---
+*(version française [ci-dessous](./2000-01-01-iso27001.md#fr-has-scalingo-iso-27001))*
```
Comment (btrd) [inline src/_posts/compliance/2000-01-01-iso27001.md:7]: Same wrong URL

See https://github.com/Scalingo/documentation#dos `If you want to insert a link to another documentation article:`

**Review by btrd — CHANGES_REQUESTED** [review]

**Comment by btrd** [comment]
Wonder why your using the FAQ format instead of a "classic" article ?

File: src/_posts/compliance/2000-01-01-hds.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
+---
+title: Hébergeur de Données de Santé - Health Data Hosting Provider 
+nav: HDS
+modified_at: 2022-09-12 00:00:00
+tags: compliance hds
+---
+
+*(version française [ci-dessous](#fr-has-scalingo-an-hds-certification))*
```
Comment (btrd) [inline src/_posts/compliance/2000-01-01-hds.md:8]: I think you want to use `-quest-ce-que-la-certification-iso-27001-` for the first french item. Same for second link

Comment (yanjost) [reply src/_posts/compliance/2000-01-01-hds.md]: You're right ! Fixed

**Review by btrd — APPROVED** [review]

File: src/_posts/compliance/2000-01-01-hds.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+## 🇬🇧 What is HDS Certification?
+
+The HDS certification aims to strengthen the protection of personal health data and to build an environment of trust around eHealth and patient monitoring.
+It is based on benchmarks including compliance with iso standards and allows certification to be issued by an accredited independent body to any structure or organization hosting health data.
+Personal health data is sensitive data. Their access is regulated by law to protect the rights of individuals. The hosting of this data must therefore be carried out under security conditions adapted to their criticality. The regulations define the terms and conditions expected.
+
+In France, all public or private organizations that host, operate the health IS, or carry out backups on behalf of a health establishment or a health third party must be HDS certified. with the exception of computer archiving services which are not affected by these obligations. Health establishments that manage their own health information system do not need to be HDS certified (unless they do so as part of a hospital group).
+
+## 🇬🇧 Is Scalingo certified for HDS ?
```
Comment (yannski) [inline src/_posts/compliance/2000-01-01-hds.md:18]:
```suggestion
## 🇬🇧 Is Scalingo certified for HDS?
```

File: src/_posts/compliance/2000-01-01-hds.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+Yes. We obtained our HDS 1.1 certification on September 12th, 2022.
+
+The certificate number is 38436, it was issued by [LNE](https://www.lne.fr/) and it is valid until September 11, 2025.
```
Comment (yannski) [inline src/_posts/compliance/2000-01-01-hds.md:22]:
```suggestion
The certificate number is 38436, it was issued by [LNE](https://www.lne.fr/en) and it is valid until September 11, 2025.
```

File: src/_posts/compliance/2000-01-01-hds.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+The HDS certification of Scalingo covers the following perimeters:
+
+1. Provision and maintenance in operational condition of physical sites to host the physical infrastructure of the information system used for processing health data;
+2. Provision and maintenance in operational condition of the physical infrastructure of the information system used for processing health data;
+3. Provision and maintenance in operational condition of the information system's application hosting platform;
+4. Provision and maintenance in operational condition of the virtual infrastructure of the information system used for processing health data;
+5. Administration and operation of the information system containing the Personal Health Data;
+6. Outsourced backups of health data.
+
+## 🇬🇧 Where can I get the official latest version of your HDS certificate ?
```
Comment (yannski) [inline src/_posts/compliance/2000-01-01-hds.md:33]:
```suggestion
## 🇬🇧 Where can I get the official latest version of your HDS certificate?
```

**Review by yannski — CHANGES_REQUESTED** [review]
Some minor EN tweaks to approve

**Review by yannski — APPROVED** [review]

**Pull request merged**